include/lbann/layers/transform/split.hpp

```diff
@@ -72,7 +72,23 @@ class split_layer : public data_type_layer<TensorDataType> {
   data_layout get_data_layout() const override { return T_layout; }
   El::Device get_device_allocation() const override { return Dev; }
 
-
+#ifdef LBANN_HAS_ONNX
+  std::string get_onnx_op_type() const override { return "Identity"; }
+
+  void fill_onnx_node(onnx::GraphProto& graph) const override {
+    for(auto const* child : this->get_child_layers()) {
+      auto* node = graph.add_node();
+      for(auto const* parent : this->get_parent_layers())
+        node->add_input(parent->get_name() + "_0");
```

Comment:
We'll run into issues if the parent has multiple children:
```suggestion
    const auto& parent = this->get_parent_layer();
    const size_t idx_in_parent = parent.find_child_layer_index(this);
    for(auto const* child : this->get_child_layers()) {
      auto* node = graph.add_node();
      node->add_input(parent->get_name() + "_" + std::to_string(idx_in_parent));
```